### DISCUSSION TITLE
按需导入NButton后没有按钮的样式。

### DISCUSSION THREAD

**Discussion by zhang1446802857** [tusen-ai/naive-ui#3988] [Q&A]
![image](https://user-images.githubusercontent.com/87363443/199905971-25f46be8-5bfa-4838-a940-71b0fab0e4d9.png)
是这样的 
![image](https://user-images.githubusercontent.com/87363443/199906028-f21236e3-018b-4de4-8907-68074bb4b2c6.png)

**Comment by zhang1446802857** [comment]
我不太清楚是少引入什么了？

**Comment by XieZongChen** [comment]
![image](https://user-images.githubusercontent.com/46394163/200162307-cf4d038d-6789-4762-8fa9-4cb28428392f.png)
https://www.naiveui.com/zh-CN/light/docs/import-on-demand

**Reply by zhang1446802857** [reply]
你发的这个，官网没有写的。我试试呢

**Reply by zhang1446802857** [reply]
对了，刚刚使用  图片组件，发现不管是本地 的还是url图片。图片都是裂开的。我用的是png和jpeg格式的图片都不行。是对图片有一定要求吗？如果采用<img src="">这样，虽然可以显示，但是大小设置不了

**Reply by XieZongChen** [reply]
裂开说明 url 请求不到图片

**Reply by XieZongChen** [reply]
> 你发的这个，官网没有写的。我试试呢

我截图截的就是官网写的，你没仔细看而已

**Reply by zhang1446802857** [reply]
这个我明白，但是我可以在网页打开。然后我尝试用本地图片代替，依然是裂开。如果是单纯用<img>是可以打开的

**Reply by XieZongChen** [reply]
那就是图片路径有问题，你从控制台看一下图片请求到底请求到什么路径了

**Reply by zhang1446802857** [reply]
请问有没有QQ群什么的 呀？我不是专门做前端的，问题可能会比较多，但……原样粘贴菜单组件都不显示……我不知道问题在哪里，我也不想因为这样的事麻烦作者，所以有没有交流群可以加一下呢？有加一个钉钉群，但是群是死的，从来没有活人…………

**Reply by XieZongChen** [reply]
> 请问有没有QQ群什么的 呀？我不是专门做前端的，问题可能会比较多，但……原样粘贴菜单组件都不显示……我不知道问题在哪里，我也不想因为这样的事麻烦作者，所以有没有交流群可以加一下呢？有加一个钉钉群，但是群是死的，从来没有活人…………

目前只有钉钉群，你可以尝试加1、2、3群，那些相对活跃
